render.go

```diff
@@ -159,8 +165,8 @@ func (r *Render) renderCompletion(completions *CompletionManager, cursorPos int)
 		r.out.CursorForward(x + width - int(r.col))
 	}
 
-	r.out.CursorUp(windowHeight)
 	r.out.SetColor(DefaultColor, DefaultColor, false)
+	return int(r.col) * windowHeight // the number of characters to go up
```

Comment:
Should we rename this function so it's obvious what it returns?

Reply:
- renderCompletionAndReturnsItsLen
- renderCompletionReturnsLen?

hmmm not a fan.. 🤔

Reply:
idk, maybe you have a better suggestion but I like renderCompletion + one comment on top more

Reply:
I've added comments on top of the two functions that returns the length to be traced back.

It somehow doesn't feel quite right to add the return to the func name

- renderCompletionAndCalcLen
- renderCompletionAndReturnsItsLen
- renderCompletionReturnsLen?

idk also not a fan of what I came up with 🤔